docs/develop/develop-office-add-ins-for-the-ipad.md

```diff
@@ -1,49 +1,47 @@
 
-Your add-in can remain as-is for Office applications that are running on other platforms. You can also serve a different UI based on the browser/device that your add-in is running on. To detect whether your add-in is running on an iPad, you can use the following APIs:
-- var isTouchEnabled = [Office.context.touchEnabled](/javascript/api/office/office.context#touchenabled)
-- var allowCommerce = [Office.context.commerceAllowed](/javascript/api/office/office.context#commerceallowed)
+> [!NOTE]
+> Your add-in can serve an alternate UI based on the device that it is running on. To detect whether your add-in is running on an iPad, you can use the following APIs:
+>
+> - var isTouchEnabled = [Office.context.touchEnabled](/javascript/api/office/office.context#touchenabled)
+> - var allowCommerce = [Office.context.commerceAllowed](/javascript/api/office/office.context#commerceallowed)
+>
+> On an iPad, `touchEnabled` returns `true` and `commerceAllowed` returns `false`.
+>
+> For information on the best UI design practices for iPad, see [Designing for iOS](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/).
 
+## Best practices for developing Office Add-ins that can run on iPad
 
-## Best practices for developing Office Add-ins for iOS and Mac
+Apply the following best practices for developing add-ins that run on iPad:
 
-Apply the following best practices for developing add-ins that run on iOS:
+-  **Develop and debug the add-in on Windows or Mac and sideload it to an iPad.**
 
-
--  **Use Visual Studio to develop your add-in.**
-
-    If you develop your add-in with Visual Studio, you can [set breakpoints and debug its code](../develop/debug-office-add-ins-in-visual-studio.md) in an Office client application running on Windows, before sideloading your add-in on the iPad or Mac. Because an add-in that runs in Office on iOS or Mac supports the same APIs as an add-in running in Office on Windows, your add-in's code should run the same way on both platforms.
+    You can't develop the add-in directly on an iPad, but you can develop and debug it on a Windows or Mac computer and sideload it to an iPad for testing. Because an add-in that runs in Office on iOS or Mac supports the same APIs as an add-in running in Office on Windows, your add-in's code should run the same way on both platforms. For details, see [Test and debug Office Add-ins](../testing/test-debug-office-add-ins.md) and [Sideload Office Add-ins on iPad and Mac for testing](../testing/sideload-an-office-add-in-on-ipad-and-mac.md).
```

Comment:
I'm a little unclear what "both platforms" represents here. Recommending update to "these platforms".
```suggestion
    You can't develop the add-in directly on an iPad, but you can develop and debug it on a Windows or Mac computer and sideload it to an iPad for testing. Because an add-in that runs in Office on iOS or Mac supports the same APIs as an add-in running in Office on Windows, your add-in's code should run the same way on these platforms. For details, see [Test and debug Office Add-ins](../testing/test-debug-office-add-ins.md) and [Sideload Office Add-ins on iPad and Mac for testing](../testing/sideload-an-office-add-in-on-ipad-and-mac.md).
```